### ISSUE TITLE
Default wizard robe holds staves, Gemsuit doesn't

### ISSUE THREAD

**Issue by Killette2** [d3athrow/vgstation13#18056]
Self explanatory

>buy staff
>put it in suit storage of default wizard suit
>buy gemsuit
>staff no longer fits in slot

**Comment by Probe1** [comment]
![image](https://user-images.githubusercontent.com/8550416/38638660-fc434b10-3d9c-11e8-992b-f0054f6d60bd.png)

**Comment by Killette2** [comment]
It's not a feature request, it's something overlooked you tard. The robe didn't always hold staffs in it's exosuit storage slot, but that got changed at a later date.  The basic wizard robe now lets you hold staffs, so why should the direct hardsuit upgrade NOT have that feature?

**Comment by Probe1** [comment]
Like I said the hardsuit gives you the ability to go into space.  Other hardsuits haven't that ability.  You want a feature.

**Comment by Killette2** [comment]
I see the base to your argument now, but I originally reported this as a bug report/oversight as it seemed to be something overlooked. I reported it along with another oversight I found pictured here.
![image](https://user-images.githubusercontent.com/5169310/38647185-03c451d8-3db9-11e8-9054-955e5f510f1b.png)

With that in mind, it seemed rather silly and tactless for you to compare my belief of an oversight to feature request in an attempt to make a mockery of my albeit many, harmless suggestions, many of which are over 2 years old and don't compare to this instance at all.

Sure I could attempt to push my first PR by slapping together what I believe would be the correct lines of code and testing it but if it is a balance issue where hardsuits give space protection and wizard robes let you hold staffs, then wouldn't it be best to know the servers opinion and add a "Discussion" tag before it is submitted as a PR and reviewed by 3 people tops?

Edit: It also seems most of my other recent post, when not sneakily filtered, also seem to be bug reports or oversights, with a few balance discussions in between.
![image](https://user-images.githubusercontent.com/5169310/38647613-304eba2a-3dbb-11e8-9638-590ccfef5e91.png)
